### ISSUE TITLE
.gitignore "fading" seems to not work in the explorer

### ISSUE THREAD

**Issue by derekcavaliero** [fabiospampinato/vscode-projects-plus#65]
First of all - thank you for this, it is very helpful.

I've been attempting to migrate my setup from Atom over to VS Code and this extension was the closest thing available to the Atom Project Manager.

One thing I have not been able to wrap my head around is that when I load a project containing a `.git` directory and a `.gitignore` - the visual "grey out" in the explorer panel isn't working. 

It's not a huge issue - but I am so used to being able to easily see that my node_modules/vendor directories are excluded by the subdued look.

I'm kind of stumped. Not sure why it wouldn't be working. I've disabled all my other extensions and the issue persists - so I _think_ its something related to this extension - but I am also not 100% sure on that.

VS Code Version:
```
Version: 1.45.1
Commit: 5763d909d5f12fe19f215cbfdd29a91c0fa9208a
Date: 2020-05-14T08:33:47.663Z
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.4.0
```

macOS Catalina 10.15.4

Thanks,
Derek

**Comment by fabiospampinato** [comment]
This is a VSCode-level issue.

I think that is governed by `files.exclude` setting, you should take a look at that.

**Comment by derekcavaliero** [comment]
Thanks for the tip.

I tried that. I am beginning to think the file watcher just isn't running whenever the projects are loaded up - if I go to File "Open" and select the same directory - the SCM automatically detects any changes to the files as you would expect...

Very odd. I'll keep digging.

**Comment by derekcavaliero** [comment]
Just following up here in case someone else runs into this issue.

Turns out VS Code was expecting the file watcher to be a different one than what was passed in:

```WARN [File Watcher (chokidar)] Watcher basePath does not match version on disk and was corrected (original: /Users/DerekCavaliero/Sites/myproject, real: /Users/derekcavaliero/Sites/myproject)```

Strange thing is that if I ran `pwd` in that directory I would get `/Users/DerekCavaliero/Sites/myproject`. So I suppose macOS has a different basePath on disk vs what is actually being used? I updated the project path in my project JSON file - rebooted VS Code and it started working... 

Very odd - but it was in fact something where the file watcher wasn't working properly.
